src/allotropy/parsers/cytiva_unicorn/structure/measurements/absorbance.py

```diff
@@ -0,0 +1,77 @@
+from abc import abstractmethod
+
+from allotropy.allotrope.models.shared.definitions.definitions import (
+    FieldComponentDatatype,
+)
+from allotropy.allotrope.schema_mappers.adm.liquid_chromatography.benchling._2023._09.liquid_chromatography import (
+    DataCubeComponent,
+    DeviceControlDoc,
+    Measurement,
+)
+from allotropy.parsers.cytiva_unicorn.constants import DEVICE_TYPE
+from allotropy.parsers.cytiva_unicorn.reader.strict_element import (
+    StrictElement,
+)
+from allotropy.parsers.cytiva_unicorn.reader.unicorn_zip_handler import (
+    UnicornZipHandler,
+)
+from allotropy.parsers.cytiva_unicorn.structure.measurements.generic import (
+    UnicornMeasurement,
+)
+from allotropy.parsers.cytiva_unicorn.structure.static_docs import (
+    StaticDocs,
+)
+from allotropy.parsers.utils.uuids import random_uuid_str
+
+
+class AbsorbanceMeasurement(UnicornMeasurement):
+    @classmethod
+    @abstractmethod
+    def get_curve_regex(cls) -> str:
+        pass
+
+    @classmethod
+    def create(
+        cls,
+        handler: UnicornZipHandler,
+        elements: list[StrictElement],
+        stat_docs: StaticDocs,
+    ) -> Measurement:
+        return Measurement(
```

Comment:
Looking at the subclass implementations, it looks to me like you could move this function to the base class, if you make functions to get `chromatogram_data_cube`, `processed_data_doc`, and `device_control_docs`